### ISSUE TITLE
Syncpack lint returns 1 error code even with no errors, and semverRange incorrect applied to package.json version

### ISSUE THREAD

**Issue by tkodev** [JamieMason/syncpack#144]
## Description

Bug 1: In a monorepo, `syncpack lint` returns a 1 if using the following configuration, despite having no errors

```
module.exports = {
  semverRange: '^',
  semverGroups: [
    {
      range: '',
      dependencyTypes: ['workspace'],
      packages: ['**'],
      dependencies: ['**']
    }
  ]
}
```

Bug 2: The reason I added semver groups for workspace is because `syncpack lint` keeps asking me to update my package version property like this `@mymonorepo/package 0.0.0 → ^0.0.0`, which is incorrect.

## Suggested Solution

- Return zero if there are zero problems returned in `syncpack lint`
- Avoid applying `semverRange`'s `^` character in monorepo's `package.json` version property, because version properties should be only explicit versions (no semver range character, format like `xx.xx.xx`)

## Help Needed

Not sure if the two bugs are related

**Comment by JamieMason** [comment]
Hi @tkodev, this was hopefully fixed in [10.6.1](https://github.com/JamieMason/syncpack/releases/tag/10.6.1), please give that a try and let me know.

**Comment by tkodev** [comment]
Thanks for the suggestion! The new version fixed the issues. Thank you!
